Projects/Team3/Smart_House/Smart_House/House.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Smart_House
+{
+    class House
+    {
+        private int _temperature;
+        private int _lux;
+        private bool _isMoving;
+        public TemperatureSensor temperatureSensor;
```

Comment:
I believe that it is better to have the List<ISensor> instead of a specific field for each sensor

Reply:
Thank you for the feedback)